project/Dependencies.scala

```diff
@@ -21,6 +21,7 @@ object Dependencies {
     // Akka is provided because Spark already includes it, and Spark's version is shaded so it's not safe
     // to use this one
     "com.typesafe.akka" %% "akka-slf4j" % "2.3.4" % "provided",
+    "com.typesafe.akka" %% "akka-cluster" % "2.3.4",
```

Comment:
Oh, hm..  good point.  Will have to investigate that, because I think kaka-cluster brings in a few other things, but maybe we can exclude akka itself.